src/test/components/JsTracer.test.js

```diff
@@ -67,11 +67,6 @@ describe('StackChart', function() {
     const { container } = renderResult;
 
     if (skipLoadingScreen) {
-      // Have React show the loading screen.
-      flushRafCalls();
```

Comment:
All these redundant calls bring no snapshot change, this is a good thing!

Reply:
Nicee!